editor/store/effects.js

```diff
@@ -145,8 +161,9 @@ export default {
 
 		let noticeMessage;
 		let shouldShowLink = true;
-		if ( ! isPublished && ! willPublish ) {
-			// If saving a non published post, don't show any notice
+
+		if ( isAutosave || ( ! isPublished && ! willPublish ) ) {
```

Comment:
When will `isAutosave` be truthy ? (Now that we've removed `REQUEST_POST_UPDATE_SUCCESS` from the `isAutosave` flow above)

Reply:
right, good catch. removed in https://github.com/WordPress/gutenberg/pull/6257/commits/f88fe54a26075e9dd46b1d9d090f8a739aa39e31